base/gmp.jl

```diff
@@ -931,7 +931,7 @@ function Base.://(x::Rational{BigInt}, y::Rational{BigInt})
         if iszero(x.num)
             throw(DivideError())
         end
-        return (isneg(x.num) ? -one(BigFloat) : one(BigFloat)) // y.num
+        return isneg(x.num) ? typemin(BigFloat) : typemax(BigFloat)
```

Comment:
```suggestion
        return (isneg(x.num) ? -one(BigInt) : one(BigInt)) // y.num
```
I don’t think the use of `BigFloat` was intentional here, so I would just restore the pre-1.6 behavior.